envoy.yaml

```diff
@@ -43,3 +30,16 @@ static_resources:
       http2_protocol_options: {}
       name: xds_cluster
```

Comment:
Can you switch the cluster name prop to line 25. it makes it easier to identify the cluster